src/sv_port.rs

```diff
@@ -197,12 +197,24 @@ fn port_nettype_ansi(
     }
 }
 
-fn port_signedness_ansi(m: &sv_parser::AnsiPortDeclaration) -> SvSignedness {
-    let signedness = unwrap_node!(m, Signing);
-    match signedness {
-        Some(RefNode::Signing(sv_parser::Signing::Signed(_))) => SvSignedness::Signed,
-        Some(RefNode::Signing(sv_parser::Signing::Unsigned(_))) => SvSignedness::Unsigned,
-        _ => SvSignedness::Unsigned,
+fn port_signedness_ansi(
+    m: &sv_parser::AnsiPortDeclaration,
+    datatype: &SvDataType,
+) -> Option<SvSignedness> {
+    match datatype {
+        SvDataType::Class => return None,
```

Comment:
Maybe I'm missing something, but I don't see the need for `return` in this function.

Reply:
Just removed it.